src/ReactiveUI/Mixins/DependencyResolverMixins.cs

```diff
@@ -28,16 +96,22 @@ public static class DependencyResolverMixins
         [SuppressMessage("Globalization", "CA1307: operator could change based on locale settings", Justification = "Replace() does not have third parameter on all platforms")]
         public static void InitializeReactiveUI(this IMutableDependencyResolver resolver)
         {
-            var extraNs = new[]
+            var possibleNamespaces = new Dictionary<RegistrationNamespace, string>
             {
-                "ReactiveUI.XamForms",
```

Comment:
The order we previously looked was used as part of our previous tests. It may be best to instead of using a dictionary to use a List. It's only 6 entries so a dictionary probably wouldn't perform better anyway.